### PR TITLE
Fix tests on MySQL

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: test/future_execution_test.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require_relative 'test_helper'
+require 'multi_json'
```
Comment (ezr-ondrej) [inline test/future_execution_test.rb:3]: This doesn't seem necessary :thinking:

**Review by ezr-ondrej — APPROVED** [review]
Apart of the uneccessary require, I like this :+1:

File: test/future_execution_test.rb (hunk 1 above)
Comment (adamruzicka) [reply test/future_execution_test.rb:3]: I needed to do this when running tests only in this file, will drop.

**Comment by ezr-ondrej** [comment]
Thanks @adamruzicka !

**Pull request merged**